SummerProject/static/src/components/styles/mainLogInFlow.css

```diff
@@ -0,0 +1,226 @@
+.login-page {
+    display: flex;
+    justify-content: flex-start;
+    max-width: 1440px;
+    padding: 0;
+    background: white;
+}
+
+.right-part {
+    align-self: center;
+    margin: auto;
+}
+
+.left-part {
+    position: relative;
+    max-width: 640px;
+    width: 100%;
+    height: 100vh;
+    overflow: hidden;
+}
+
+.logo {
+    position: absolute;
+    top: 0;
+    left: 0;
+    z-index: 1;
+    background: #D72130;
+    width: 245px;
+    height: 83px;
+    text-align: center;
+}
+
+a.logo-link {
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    width: 50%;
+    height: 30%;
+    margin: auto;
+    text-decoration: none;
+    font-family: sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 22px;
+    line-height: 28px;
+    color: #FFFBFB;
+}
+
+.right-part p {
+    margin-bottom: 30px;
+}
+
+.log-in {
+    position: absolute;
+    top: 0;
+    right: 0;
+    display: flex;
+    width: 375px;
+    height: 83px;
+    text-align: center;
+    justify-content: center;
+    z-index: 1;
+
+}
+
+a.login-link {
+    position: absolute;
+    left: 15%;
+    top: 26%;
+    bottom: 6.67%;
+    font-family: sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 12px;
+    line-height: 17px;
+    text-align: center;
+    color: #596377;
+}
+
+a.login-link-mobile{
+    font-family: sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 12px;
+    line-height: 17px;
+    color: #FFFFFF;
+}
+
+.right-part h1 {
+    font-family: sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 24px;
+    line-height: 33px;
+    color: #000000;
+}
+
+.right-part p {
+    font-family: sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 12px;
+    line-height: 25px;
+    color: #434C5D;
+}
+
+.log-in .btn {
+    position: absolute;
+    left: 55.93%;
+    right: 0;
+    top: 15%;
+    bottom: 8.57%;
+    display: inline-block;
+    width: 30%;
+    height: 32px;
+    font-family: sans-serif;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 600;
+    line-height: 26px;
+    color: #D72130;
+    text-align: center;
+    text-transform: capitalize;
+    text-decoration: none;
+    cursor: pointer;
+    background-color: white;
+    border: 1px solid #D72130;
+    border-radius: 0;
+    box-sizing: border-box;
+}
+
+.log-in .btn:hover, .log-in .btn:active {
+    color: white;
+    background: #D72130 !important;
+}
+
+.btn-primary {
+    display: inline-block;
+    width: 400px;
+    height: 42px;
+    color: white;
+    text-transform: capitalize;
+    cursor: pointer;
+    background-color: #D72130;
+    border: 1px solid #D72130;
+    border-radius: 0;
+    box-sizing: border-box;
+}
+
+
+.btn-primary:hover, .btn-primary:active {
+    background: white !important;
```

Comment:
Are you sure that you need `!important` here?

Reply:
Yeah, without !important it doesn't work correctly

Reply:
Ok, just wanted make sure that you really need it.